dotcom-rendering/src/server/dev-index.html

```diff
@@ -648,8 +648,13 @@ <h2>Test Articles By Element (Missing)</h2>
 				// All interactive pages include this in the URL
 				if (url.includes('/ng-interactive/')) return 'Interactive';
 				// We're looking for the date string, e.g /2022/jul/30
-				else if (url.match(/[0-9]{4}\/[a-z]{3}\/[0-9]{2}\/.+/))
+				else if (
+					url.match(/(\/\d{4}\/[a-z]{3}\/\d{2}\/|\/[a-z-]+\/\d+$)/)
```

Comment:
Regexes give me the fear.

Could we make them a little easier to read by using named values? I haven't tested this but something along the lines of:

```suggestion
          const yyyyMmmDD = new RegExp(\/\d{4}\/[a-z]{3}\/\d{2}\/);
          const crosswords = new RegExp(\/[a-z-]+\/\d+$)/);
					else if (url.test(yyyyMmmDD) || url.test(crosswords))
```